packages/pwa/app/components/_app/index.jsx

```diff
@@ -163,26 +190,36 @@ const App = (props) => {
 
                             {/* Urls for all localized versions of this page (including current page)
                             For more details on hrefLang, see https://developers.google.com/search/docs/advanced/crawling/localized-versions */}
-                            {getSupportedLocalesIds().map((locale) => (
+                            {getSupportedLocalesIds(site?.l10n?.supportedLocales).map((locale) => (
                                 <link
                                     rel="alternate"
                                     hrefLang={locale.toLowerCase()}
-                                    href={`${appOrigin}${getUrlWithLocale(locale, {location})}`}
+                                    href={`${appOrigin}${getUrlWithLocale(locale, {
+                                        location
+                                    })}`}
                                     key={locale}
                                 />
                             ))}
                             {/* A general locale as fallback. For example: "en" if default locale is "en-GB" */}
                             <link
                                 rel="alternate"
                                 hrefLang={defaultLocale.slice(0, 2)}
-                                href={`${appOrigin}${getUrlWithLocale(defaultLocale, {location})}`}
+                                href={`${appOrigin}${getUrlWithLocale(defaultLocale, {
+                                    location
+                                })}`}
                             />
                             {/* A wider fallback for user locales that the app does not support */}
                             <link rel="alternate" hrefLang="x-default" href={`${appOrigin}/`} />
                         </Seo>
 
                         <ScrollToTop />
-
+                        <script
+                            id="app-config"
+                            type="application/json" // Not executable
+                            dangerouslySetInnerHTML={{
+                                __html: JSON.stringify(config)
+                            }}
+                        />
```

Comment:
Just a reminder once we merge this [PR](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/390) you won't have to set a script value like this on your own.

Reply:
This might actually come your way via this [PR](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/421) instead